src/cr/cube/matrix/measure.py

```diff
@@ -8,7 +8,11 @@
 from scipy.stats import t
 
 from cr.cube.matrix.cubemeasure import CubeMeasures
-from cr.cube.matrix.subtotals import SumSubtotals, NanSubtotals, NoneSubtotals
+from cr.cube.matrix.subtotals import (
+    SumSubtotals,
+    NanSubtotals,
+    NoneSubtotals,
+)
```

Comment:
This will fail Black on test. If you remove the last comma (after `NoneSubtotals`) it should reformat back to a single line. That one-way change is a gotcha in Black on Python 3. It puts it in if its missing, but doesn't take it out if it will fit on one line again once you make a deletion.